src/server/pfs/pfs.go

```diff
@@ -281,10 +298,18 @@ func (e ErrSquashWithoutChildren) Error() string {
 	return fmt.Sprintf("cannot squash a commit that has no children as that would cause data loss, use the drop operation instead: %s", e.Commit)
 }
 
+func (e ErrDeleteWithDependentCommitSets) Error() string {
+	return fmt.Sprintf("the commit sets %v, cannot be squashed in isolation. To delete them, also squash: %v", e.RequestedDeleteCommitSets, e.MinimalCommitSets)
```

Comment:
Should be `"the commit sets %v, cannot be squashed in isolation; to delete them, also squash: %v"` so that it composes well.

Should the error messages themselves be this explicit, or should `pachctl` turn `ErrDeleteWithDependentCommitSets` into something more informative for an end-user?

Reply:
in this case in particular, it helps to be this specific because the computation to calculate e.MinimalCommitSets is not trivial for the client